.github/workflows/tests.yml

```diff
@@ -315,6 +264,7 @@ jobs:
             typegate/tests/runtimes/python/*.ts \
             dev/deps.ts \
             dev/utils.ts
+
           deno --unstable-worker-options --unstable-net coverage ./coverage --lcov > coverage.lcov
```

Comment:
also this? or in a wider task - gha related?

Reply:
Postponed until we have container ci task runners.